static/sw.js

```diff
@@ -12,28 +20,78 @@ this.addEventListener("install", event => {
       console.log("Opened cache");
       return cache.addAll(URLS);
     })
-    .catch(err => { 
+    .catch(err => {
       console.log(err);
       throw err;
     })
   );
 });
 
+this.addEventListener("sync", event => {
```

Comment:
❗️
По файлу скачут кавычки + в воркере принято обращаться к контексту через `self`
```suggestion
this.addEventListener('sync', event => {
```